### DISCUSSION TITLE
Welcome to 100x-orchestrator Discussions!

### DISCUSSION THREAD

**Discussion by aj47** [aj47/100x-orchestrator#31] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by heijligers** [comment]
thank you for the initiative.
For my workflow I am very much exploring different architectures, as opposed to implementing a list of features.
I've been looking at AIDEML for inspiration on how to manage this, together with microsoft's Trace library that might lead to some interesting metaoptimizations.
I'll report back when I have any substantive feedback.

**Reply by aj47** [reply]
Thank you for the engagement!

someone in my community also linked microsofts magnetic-one which you might be interested in;
https://www.microsoft.com/en-us/research/articles/magentic-one-a-generalist-multi-agent-system-for-solving-complex-tasks/

**Comment by heijligers** [comment]
thanks for the breadcrumbs

On Fri, 20 Dec 2024 at 00:10, AJ ***@***.***) ***@***.***>
wrote:

> Thank you for the engagement!
>
> someone in my community also linked microsofts magnetic-one which you
> might be interested in;
>
> https://www.microsoft.com/en-us/research/articles/magentic-one-a-generalist-multi-agent-system-for-solving-complex-tasks/
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/aj47/100x-orchestrator/discussions/31#discussioncomment-11622796>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAX7BV6225VFW7HLHDUMPYD2GNG5VAVCNFSM6AAAAABTIDOML2VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCNRSGI3TSNQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***
> com>
>


-- 
Bjorn Heijligers
+31620106733
